refinery/ui/source/js/data-set-visualization/partials/visualization.html

```diff
@@ -0,0 +1,122 @@
+<div ng-switch on="$ctrl.visLoadingFlag">
+
+  <span ng-switch-when="EMPTY">
+    <em>No visualizations performed. </em>
+  </span>
+  <span ng-switch-when="LOADING">
+      <span class="refinery-spinner analyses-view"></span>
+      &nbsp;<em>Retrieving visualization list.</em>
+  </span>
+  <span ng-switch-default>
+    <div class="row analysis-header">
+      <div class="col-md-1 analysis-icon" id="header">
+        <a class="analysis-header">
+          <strong>Status</strong>
+          <span>
+             <i ng-class="analysisIcon"></i>
+          </span>
+        </a>
+      </div>
+      <div class="col-md-5 analysis-name">
+        <a class="analysis-header" ng-click="order('name')">
+          <strong>Name</strong>
+        </a>
+      </div>
+
+      <div class="col-md-1">
+       <a class="analysis-header">
+         <strong>Creation Time</strong>
+         </span>
+       </a>
+      </div>
+      <div class="col-md-2">
+        <strong>Owner Name</strong>
+      </div>
+      <div class="col-md-2">
```

Comment:
What's the empty div for? If it's filler, I think the column count is still only 11, not 12.